.eslintrc.yml

```diff
@@ -2,6 +2,12 @@ extends: airbnb-base
 plugins:
   - import
 rules:
+  no-underscore-dangle:
+  - error
+  - allowAfterThis: true
+  space-before-function-paren:
+  - error
+  - named: always
```

Comment:
We are using semistandard, which is based on standard, which uses eslint. They (standard) don't allow  [1] any customization to their set of rules.

Since we are using eslint too, we can extend the standard rules [2], sprinkle some semicolons and modify to allow `function likeThis()`

[1] https://github.com/feross/standard#i-disagree-with-rule-x-can-you-change-it
[2] https://github.com/feross/eslint-config-standard